pkg/config/config.go

```diff
@@ -1052,6 +1063,15 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 		return err
 	}
 
+	containerMinMemory, err := units.RAMInBytes(c.DefaultContainerMinMemory)
+	if err != nil {
+		return fmt.Errorf("wrong value for default-container-min-memory: %q error: %s", c.DefaultContainerMinMemory, err)
+	}
+
+	if containerMinMemory < defaultContainerMinMemoryLimit {
+		return fmt.Errorf("set memory limit %s too low; should be at least %s", c.DefaultContainerMinMemory, units.BytesSize(defaultContainerMinMemoryLimit))
+	}
```

Comment:
Do we need this check and the `initializeRuntimeContainerMemory()` if we have `GetContainerMinMemory()`?